Client/core/CSettings.cpp

```diff
@@ -1238,7 +1252,7 @@ void CSettings::CreateGUI()
     vecTemp.fX -= fComboWidth + 15;
 
     // Description label
-    vecTemp.fY = 354 + 10;
+    vecTemp.fY += 10;
```

Comment:
why was it changed?

Reply:
354 became wrong because I added a new setting, moving everything down. This is actually the correct way of determining the last fY, just incrementing 10 after the previous setting line